web/directory/management/commands/parse_coop_csv.py

```diff
@@ -122,6 +124,17 @@ def get_address_pks(file_path, city_pks):
         i=1
         address_pks = dict()
         for row in input_file:
+
+            # New code for testing whether the address can be public
+            try:
+                adrs_pub = row.get('ent-adrs-pub',"00").strip().encode("utf-8", 'ignore').decode("utf-8")
+            except KeyError:
+                adrs_pub = row['ent-adrs-pub'].strip().encode("utf-8", 'ignore').decode("utf-8")
```

Comment:
Thanks for this update!  One more question -- now that you have changed line 130, do we even need the try/except block?  That is, can we delete lines 129-132 and only leave line 130?